README.md

```diff
@@ -77,11 +77,11 @@ OBS, Streamlabs, Restream and many others have been used with Owncast. [Read mor
 ## Building from Source
 
 1. Ensure you have pre-requisites installed.
-    - [gcc compiler](https://gcc.gnu.org/install/download.html)
+    - C compiler, such as [GCC compiler](https://gcc.gnu.org/install/download.html) or a [Musl-compatible compiler](https://musl.libc.org/)
     - [ffmpeg](https://ffmpeg.org/download.html)
-1. Install the [Go toolchain](https://golang.org/dl/) (1.16 or above).
+1. Install the [Go toolchain](https://golang.org/dl/) (1.16 or above). If your package manager has an older version of go, then you can install and use a newer version with the [Go Version Manager](https://github.com/moovweb/gvm).
```

Comment:
I fear this makes Go Version Manager look like an additional tool that needs to be installed to build Owncast, when in reality you can just download an updated version from the provided link.